package.json

```diff
@@ -78,6 +79,8 @@
     "lint": "prettier --list-different '**/*.{js,css,md,json,yaml,yml}'",
     "format": "prettier --write '**/*.{js,css,md,json,yaml,yml}'",
     "test:ci": "npm run lint -s && npm run dist -s && flow && npm run test:coverage -s -- --outputFile build/test-results.json --json",
-    "prepublishOnly": "npm run dist -s"
+    "prepublishOnly": "npm run dist -s",
+    "docs:generate": "./node_modules/.bin/documentation build src/** -f html -o docs",
+    "docs:serve": "./node_modules/.bin/documentation serve src/**"
```

Comment:
I suppose this could use `documentation` directly instead of `node_modules/.bin/documentation`, since we are inside npm scripts.

Reply:
✅